src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -834,6 +884,40 @@ void Pane::SplitHorizontal(const GUID& profile, const TermControl& control)
     _Split(SplitState::Horizontal, profile, control);
 }
 
+// Method Description:
+// - Determines whether the pane can be split.
+// Arguments:
+// - splitType: what type of split we want to create.
+// Return Value:
+// - True if the pane can be split. False otherwise.
+bool Pane::_CanSplit(SplitState splitType)
+{
+    const bool changeWidth = _splitState == SplitState::Vertical;
+
+    const Size actualSize{ gsl::narrow_cast<float>(_root.ActualWidth()),

```

Comment:
Should this be `gsl::narrow_cast` or `gsl::narrow`? I honestly don't know, but I know `gsl::narrow_cast` can throw an exception and `gsl::narrow` doesn't.

Reply:
The other way around. `gsl::narrow_cast` means shrink it and I don't care if it doesn't fit exactly.

`gsl::narrow` means if it didn't fit, throw an exception.

Reply:
@carlos-zamora Since `double`->`float` is only going to cause a precision loss, I'm assuming that's preferable to keep using `gsl::narrow_cast`?